renovate.sh

```diff
@@ -9,7 +9,7 @@ do
  --token="${RENOVATE_TOKEN}" \
  --git-author="OpenStack K8s CI <openstack-k8s@redhat.com>" \
  --log-file-level=debug \
- --allowed-post-upgrade-commands="^make manifests generate,^make gowork,^make gotidy,^go mod tidy" \
+ --allowed-post-upgrade-commands="^make manifests generate,^make gowork,^make gotidy,^go mod tidy,^make tidy" \
```

Comment:
I used `make gowork` in lib-common.

Reply:
Yeah, I saw that lib-common has `make gotidy` target that basically doing what @amolkahat added in service operators as `make tidy`. I can live with this discrepancy.

Reply:
https://github.com/openstack-k8s-operators/lib-common/pull/233 could then remove `make gotidy`

Reply:
I'm OK the remove make gotidy from lib-common. After that we can drop `make gotidy` from the --allowdd-post-upgrade-commands list

Reply:
https://github.com/openstack-k8s-operators/lib-common/pull/233 is merged now so I updated this to remove make gotidy from the allow list